cmd/rook/block/map.go

```diff
@@ -131,20 +127,34 @@ func mountBlock(name, poolName, mountPoint, rbdSysBusPath string, c client.RookR
 		return "", err
 	}
 
-	// format the device with a default file system
-	if err := sys.FormatDevice(devicePath, executor); err != nil {
-		return "", fmt.Errorf("failed to format device %s: %+v", devicePath, err)
+<<<<<<< HEAD:cmd/rook/block/map.go
+	successMessage := fmt.Sprintf("succeeded mapping image %s on device %s", name, devicePath)
+
+	if formatRequested {
+		// format the device with a default file system
+		if err := sys.FormatDevice(devicePath, executor); err != nil {
+			return "", fmt.Errorf("failed to format device %s: %+v", devicePath, err)
```

Comment:
We have partial success, and I'd like to see the successMessage added to the error message so the user sees both what succeeded and what failed.